src/typescript-ast-nodes.js

```diff
@@ -0,0 +1,131 @@
+module.exports = function(fork) {
```

Comment:
Adding all typescript nodes instead of just the for missing ones here probably fixes the workaround in fast-path for typescript nodes.